README.md

```diff
@@ -58,19 +58,17 @@ juju config docker-registry \
 
 ### Proxied Registry
 
-This charm supports an `http` proxy relation that allows operators to
+This charm supports `http` and `tcp` proxy relation that allows operators to
```

Comment:
I don't think this is true.  This PR is adding support for registry<->proxy via TLS iff valid certs are configured on both ends.  IOW, this is "SSL termination" in haproxy speak.  We are not supporting "SSL passthru" here, which is what would be implied by a `tcp` relation.

Also I think the original messaging was more that the [http interface](https://github.com/CanonicalLtd/docker-registry-charm/blob/master/metadata.yaml#L17) is used by the charm, versus any implication of how haproxy would communicate with the registry.